transformers/modeling_tf_utils.py

```diff
@@ -494,7 +494,7 @@ def call(self, inputs, training=False):
 def shape_list(x):
     """Deal with dynamic shape in tensorflow cleanly."""
     static = x.shape.as_list()
-    dynamic = tf.shape(x)
+    dynamic = shape_list(x)
```

Comment:
@thomwolf Looks like this one was not intended

Reply:
Ouuups!